client/src/components/Notices/Company.js

```diff
@@ -54,8 +76,10 @@ const _Company = ({item, toggledOn, toggle}: Props) => {
 }
 
 export default compose(
-  withState('toggledOn', 'toggle', false),
+  withState('showSupplierInfo', 'toggleSupplier', false),
+  withState('showCustomerInfo', 'toggleCustomer', false),
   withHandlers({
-    toggle: ({toggle}) => (e) => toggle((current) => !current),
+    toggleSupplier: ({toggleSupplier}) => () => toggleSupplier((current) => !current),
+    toggleCustomer: ({toggleCustomer}) => () => toggleCustomer((current) => !current),
```

Comment:
I've seen this everywhere through the app and it starts to really bother me. There are three occurences of `toggleCustomer` in this line and it's two different things - one of them is state updater with one parameter and second is a handler with no parameter. When I saw that for the first time, I stared at it for few minutes and had no idea what's going on. I would really appreciate if we used separate names for handlers (`toggleCustomer` is ok, because it's toggling the value) and for state setters (this should be called `setCustomer`, because it's accepting a parameter which is the set as the value in state). I perfectly understand that you did it like this because it's done like that everywhere else, but let's strive for a better codebase together :muscle: :slightly_smiling_face: